nrcatalogtools/lvc.py

```diff
@@ -302,60 +332,62 @@ def get_ref_time_from_ref_freq(H5File, ref_freq):
     """
     from scipy.interpolate import interp1d
 
-    time, freq = H5File.attrs["Omega-vs-time"]
+    time, freq = h5_file.attrs["Omega-vs-time"]
 
-    RefTime = interp1d(freq, time, kind="cubic")[ref_freq]
+    ref_time = interp1d(freq, time, kind="cubic")[ref_freq]
 
-    return RefTime
+    return ref_time
 
 
 def check_nr_attrs(
-    MetadataObject, ReqAttrs=["LNhatx", "LNhaty", "LNhatz", "nhatx", "nhaty", "nhatz"]
+    sim_metadata_object,
+    req_attrs=["LNhatx", "LNhaty", "LNhatz", "nhatx", "nhaty", "nhatz"],
 ):
-    """Check if the NR h5 file or a metadata dictionary
-        contains all the attributes required.
-
-    Parameters
-    ----------
-    MetadataObject : h5 file object, dict
-                     The NR h5py file handle or metadata.
-    ReqAttrs : list
-               A list of attribute keys.
-    Returns
-    -------
-    Present : bool
-              Whether or not all specified attributes are present.
-    AbsentAttrs : list
-                 The attributes that are absent.
+    """Check if the NR h5 file or a simulation metadata dictionary
+            contains all the attributes required.
+
+        Parameters
+        ----------
+        sim_metadata_object : h5 file object, dict
+                         The NR h5py file handle or simulation metadata.
+        req_attrs : list
+                   A list of attribute keys.
+        Returns
+        -------
+        present : bool
+                  Whether or not all specified attributes are present.
+    h = self.interpo    absent_attrs : list
+                     The attributes that are absent.
```

Comment:
There is something wrong here. Why is there a `h=self.interpo`?

Reply:
seems like a copy-pasta error

Reply:
True. I'm having issues with mouse!